tools/dev/create_module_docs.rb

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+
+##
+#
+# This generates documentation for a module based on user prompts.
+#
+##
+
+require 'fileutils'
+
+def prompt(*args)
+    print(*args)
+    gets
+end
+
+#
+# Functions we can't live without
+#
+
+def print_status(msg='')
+  puts "[*] #{msg}"
+end
+
+def print_error(msg='')
+  puts "[-] #{msg}"
+end
+
+# The path for all module docs
+DOCS_PATH = File.join('..', '..', 'documentation', 'modules')
```

Comment:
We should probably grab the `MANUAL_BASE_PATH` from the pull request finder so that this tool is always in sync.

Reply:
Can you clarify what you mean by in sync?

Reply:
If we change where module documentation is stored and update the library, we won't need to remember to update this script since the constant will be defined in one place.

Reply:
Ah okay makes sense